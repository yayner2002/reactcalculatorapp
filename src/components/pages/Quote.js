import React from 'react';
import '../styles/Quote.css';

function quote() {
  return (
    <div className="quoteContainer">
      <h1>Best Quotes of maths</h1>
      <h4>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Error fuga
        deleniti facilis ipsa odit, veniam hic quisquam optio nihil quas qui
        beatae dolores recusandae molestias sapiente expedita impedit, dolore
        consectetur.
      </h4>
    </div>
  );
}

export default quote;
