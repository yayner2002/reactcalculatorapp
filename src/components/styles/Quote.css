.quoteContainer {
  width: 90%;
  margin: 2rem auto;
  box-sizing: border-box;
}
