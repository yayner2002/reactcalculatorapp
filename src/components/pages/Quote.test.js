import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('correctlly renders a component', () => {
  test('should render quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
