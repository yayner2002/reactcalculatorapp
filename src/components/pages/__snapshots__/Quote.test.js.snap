// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctlly renders a component should render quote component 1`] = `
<div
  className="quoteContainer"
>
  <h1>
    Best Quotes of maths
  </h1>
  <h4>
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Error fuga deleniti facilis ipsa odit, veniam hic quisquam optio nihil quas qui beatae dolores recusandae molestias sapiente expedita impedit, dolore consectetur.
  </h4>
</div>
`;
